Runtime/Triangulator.cs

```diff
@@ -173,187 +234,217 @@ public class InputData
-
-            dependencies = new DelaunayTriangulationJob(this).Schedule(dependencies);
-            dependencies = Input.ConstraintEdges.IsCreated ? new ConstrainEdgesJob(this).Schedule(dependencies) : dependencies;
-            dependencies = Input.ConstraintEdges.IsCreated && (Settings.RestoreBoundary || Input.HoleSeeds.IsCreated) ? new PlantingSeedsJob(this).Schedule(dependencies) : dependencies;
-            dependencies = Settings.RefineMesh ? new RefineMeshJob(this).Schedule(dependencies) : dependencies;
-
-            dependencies = Settings.Preprocessor switch
-            {
-                Preprocessor.PCA => SchedulePCAInverseTransformation(dependencies),
-                Preprocessor.COM => ScheduleLocalToWorldTransformation(dependencies),
-                Preprocessor.None => dependencies,
-                _ => throw new NotImplementedException()
-            };
-
-            return dependencies;
-        }
-
-        private JobHandle SchedulePCATransformation(JobHandle dependencies)
-        {
-            tmpInputPositions = Input.Positions;
-            Input.Positions = localPositions.AsDeferredJobArray();
-            if (Input.HoleSeeds.IsCreated)
-            {
-                tmpInputHoleSeeds = Input.HoleSeeds;
-                Input.HoleSeeds = localHoleSeeds.AsDeferredJobArray();
-            }
-
-            dependencies = new PCATransformationJob(this).Schedule(dependencies);
-            if (tmpInputHoleSeeds.IsCreated)
-            {
-                dependencies = new PCATransformationHolesJob(this).Schedule(dependencies);
-            }
-            return dependencies;
-        }
-
-        private JobHandle SchedulePCAInverseTransformation(JobHandle dependencies)
-        {
-            dependencies = new PCAInverseTransformationJob(this).Schedule(this, dependencies);
-
-            Input.Positions = tmpInputPositions;
-            tmpInputPositions = default;
-
-            if (tmpInputHoleSeeds.IsCreated)
-            {
-                Input.HoleSeeds = tmpInputHoleSeeds;
-                tmpInputHoleSeeds = default;
-            }
-
-            return dependencies;
-        }
-
-        private JobHandle ScheduleWorldToLocalTransformation(JobHandle dependencies)
-        {
-            tmpInputPositions = Input.Positions;
-            Input.Positions = localPositions.AsDeferredJobArray();
-            if (Input.HoleSeeds.IsCreated)
-            {
-                tmpInputHoleSeeds = Input.HoleSeeds;
-                Input.HoleSeeds = localHoleSeeds.AsDeferredJobArray();
-            }
-
-            dependencies = new InitialLocalTransformationJob(this).Schedule(dependencies);
-            if (tmpInputHoleSeeds.IsCreated)
-            {
-                dependencies = new CalculateLocalHoleSeedsJob(this).Schedule(dependencies);
+        [BurstCompile]
+        struct TriangulationJob: IJob {
```

Comment:
Please move this to `#region Jobs`. 

PS
After merging this PR, other jobs can be replaced with methods, which should simplify package even further :)